.github/workflows/action-pypi-build-and-deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: PyPi Build and Deploy 🐍📦
+
+on:
+  release:
+    types: [published]
+  # use this for testing
+  push:
+    branches:
+      - master
+
+jobs:
+  build-n-publish:
+    name: Build and publish ESMValTool on PyPi
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install pep517
+        run: >-
+          python -m
+          pip install
+          pep517
+          --user
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          pep517.build
+          --source
+          --binary
+          --out-dir dist/
+          .
+      # Publish on Test PyPi
+      - name: Publish distribution 📦 to Test PyPI
+        if: startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.test_pypi_password }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
This would need to be removed before merging. Also, if you want to test without making a release, line 37 would need to be removed.

Reply:
OK, we can just do the full PyPi dance and forget about the Test bit

Reply:
yeah just asked @mattiarighi to add the tokens to ESMValTool :+1: